### PR TITLE
 Profile sapling key table modifications and paging

### PR DESCRIPTION
**Pull request by isabeltomb**
- Adds paging to the key table by using the material-ui TablePagination React component
- Adds labels to buttons that show when hovering the cursor
- Fixes the action options to stay in the action column. Previously, if the browser window was smaller than the table width, requiring horizontal scrolling, the expanded action options didn't stay in the action column
- Fixes the table to allow the action options for the last key in the table to expand beyond the bottom border of the table
- Adjusts the table navigation padding to align with column names in the key table

**Testing:**
1. Run the splinter-ui using either the biome docker-compose or set the necessary environment variables and use the oauth docker-compose
`docker-compose -f docker-compose-biome.yaml up --build`
OR
`docker-compose -f docker-compose-oauth.yaml up --build`

2. In a browser navigate to `http://localhost:3030`

3. Register/log in and select the profile tab in the left nav bar

4. Add at least 11 keys to the table to test paging

5. Hover cursor over buttons to see labels

### CODE REVIEW

**Review by shannynalayna — COMMENTED** [review]
Could we set it so that the active key is displayed first in the table? I noticed when updating any key it puts it to the end of the list (which may be a different page). I would expect the activated key to be displayed first (unless being sorted by something) which these don't seem to be

**Comment by isabeltomb** [comment]
@shannynalayna added a commit to this PR to sort the keys in the table to always keep the active key in the first row of the table

**Review by shannynalayna — DISMISSED** [review]

File: saplings/profile/src/Profile.js [hunk 1]
```diff
@@ -83,7 +83,20 @@ export function Profile() {
     setModalActive(true);
   };
 
+  const sortKeysActive = (allKeys, publicKey) => {
+    const activeKey = allKeys.find(key => key.public_key === publicKey);
+    const keyIndex = allKeys.findIndex(key => key.public_key === publicKey);
```
Comment (peterschwarz) [inline saplings/profile/src/Profile.js:87]: Looks like you could just do this search, and then directly index for the activeKey:
```javascript
const keyIndex // same as this line
const activeKey = allKeys[keyIndex];
```

**Review by peterschwarz — CHANGES_REQUESTED** [review]

File: saplings/profile/src/Profile.js (hunk 1 above)
Comment (isabeltomb) [reply saplings/profile/src/Profile.js:87]: Fixed

**Review by peterschwarz — APPROVED** [review]

**Review by shannynalayna — APPROVED** [review]

**Pull request merged**